config/config.extensions.js

```diff
@@ -1,6 +1,30 @@
+const yaml = require('js-yaml');
+const path = require('path');
+const fs = require('fs');
+
+const SERVERS_YML = path.join(__dirname, '..', 'language-servers.yml');
+
+function getAvailableLanguageServers () {
```

Comment:
I am a little wary about this. It seems to be assuming that any servers defined in `language-servers.yml` are available but since that file is checked in to git, it seems likely that this will end up with invalid language servers.

Reply:
Yep. can it be solved by moving `language-servers.yml` to the deployment directory and using the config in production mode only?